### PR TITLE
Markdown

### PR DESCRIPTION
**Pull request by jiwalker-usgs**
I ended up messing around with the page and section publishers for handling dependencies.  This will hopefully not affect anything adversely, but I can see that this is getting a bit unruly.  Do you see any issues or improvements to this approach @lindsaycarr @wdwatkins?

### CODE REVIEW

**Comment by jiwalker-usgs** [comment]
fix for #47

File: R/export.R [hunk 1]
```diff
@@ -73,6 +75,19 @@ export.css <- function(viz) {
   return(file)
 }
 
+#' Default export, defaults to not export
```
Comment (wdwatkins) [inline R/export.R]: I'm a bit confused by this description

Comment (jiwalker-usgs) [inline R/export.R]: This is the default export method.  It defaults to not export the contents to the target directory and requires either an override or explicitly setting `export: true` on the object.

File: R/publish.R [hunk 2]
```diff
@@ -65,8 +70,18 @@ publish.section <- function(viz) {
   checkRequired(viz, required)
 
   # TODO Watch out for cyclic depends
-  dependencies <- lapply(viz[['depends']], publish)
+  dependencies <- as.list(viz[['depends']])
+  dependencies <- lapply(dependencies, function(x) {
+    expanded.dep <- NULL
```
Comment (wdwatkins) [inline R/publish.R]: Looks like you could break this out as a named function since it's also used above?

**Comment by wdwatkins** [comment]
Looks ok to me, so the main difference is that the output of all the published dependencies is being returned inside the `lapply`?  In addition to the `cat` at the end?

File: R/publish.R (hunk 2 above)
Comment (jiwalker-usgs) [inline R/publish.R]: good idea, I thought these might be different, but they ended up equivilent.

**Comment by jiwalker-usgs** [comment]
The main difference is that it is doing publish and/or readData so it can pick up on dependencies that are nested.  It also introduces the dot notation for nested dependencies, which is hopefully easy to follow.  Added to example https://github.com/USGS-VIZLAB/example/pull/12.

**Comment by lindsaycarr** [comment]
:100: !!!!

**Pull request merged**